Apps/MineOS/docker-compose.yml

```diff
@@ -0,0 +1,165 @@
+            
+      ports:
+        - container: "8443"
+          description:
+            en_us: WebUI HTTP Port
+          protocol: tcp
+
+        - container: "25565"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25566"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25567"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25568"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25569"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25570"
+          description:
+            en_us: Minecraft listening Port (TCP)
+          protocol: tcp
+
+        - container: "25565"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25566"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25567"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25568"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25569"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+        - container: "25570"
+          description:
+            en_us: Minecraft listening Port (UDP)
+          protocol: udp
+
+      volumes:
+        - container: /var/games/minecraft
+          description:
+            en_us: MineOS directory.
+x-casaos:
+  architectures:
+    - amd64
+    - arm
+    - arm64
+  main: mineos
+  author: Chàra Rose
+  category: Games
+  description:
+    en_us: MineOS is a server front-end to ease managing Minecraft administrative tasks. This iteration using Node.js aims to enhance previous MineOS scripts (Python-based), by leveraging the event-triggering, asyncronous model of Node.JS and websockets.
+    es_es: MineOS es una interfaz de servidor para facilitar la gestión de las tareas administrativas de Minecraft. Esta iteración que utiliza Node.js tiene como objetivo mejorar los scripts anteriores de MineOS (basados en Python), aprovechando el modelo asincrónico de activación de eventos de Node.JS y websockets.
+  developer: hexparrot
+  icon: https://cdn.jsdelivr.net/gh/IceWhaleTech/CasaOS-AppStore@main/Apps/MineOS/icon.png
+  tagline:
+    en_us: Free and easy to use Minecraft server management tool.
+    es_es: Herramienta de administración de servidores de Minecraft gratuita y fácil de usar.
+  tips: {}
+  title:
+    en_us: MineOS
+  index: /admin/index.html
```

Comment:
should add `scheme: https` - otherwise, web ui points to `http://...` which returns empty response

Reply:
OR - use plain HTTP instead of HTTPS - it's OK if it is running inside a home network ;)